### PR TITLE
add Fever

### PR DESCRIPTION
**Pull request by rettgerst**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,6 +164,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [Creaky Coot](https://github.com/piero-la-lune/Creaky-Coot) - Minimalist and responsive RSS reader and links saver
   * [Feedbin](https://feedbin.me) - a simple, fast and nice looking RSS reader. ([Source code](https://github.com/feedbin/feedbin))
   * [FeedHQ](https://feedhq.org/) - FeedHQ is a web-based feed reader ([Source code](https://github.com/feedhq/feedhq))
+  * [Fever](http://feedafever.com/) - Self-hosted feed aggregator that can sort items by popularity
```
Comment (nodiscc) [inline README.md]: Please add `⊘ Proprietary` after the software name

**Comment by nodiscc** [comment]
I can't find info on how Fever guesses items popularity. Is it all local or does it send/receive info from a third-party?

**Comment by rettgerst** [comment]
I'm not intimately familiar with the service, but if I understand correctly it's all done locally. basically it guesses which of your items are talking about the same subject and then groups them by subject. subjects that are mentioned by more feeds that you subscribe to are "hotter".

[this review](http://www.macstories.net/reviews/fever-ashes-ipad/) sheds some light on how it works.

**Comment by nodiscc** [comment]
I'd really like if someone could contact the developer to clarify whether if it sends any kind of data to a third-party (except for fetching/displaying feeds of course) - the `⛔ Anti-features` label would then apply. Otherwise this looks interesting, thanks @rettgerst

**Comment by jakoubek** [comment]
Fever doesn't send any data to a third-party (I've been using it for years now). It just groups the topics.

**Comment by nodiscc** [comment]
ok. this is just missing `PHP`

**Pull request merged**